src/components/nav/nav.js

```diff
@@ -0,0 +1,54 @@
+export default class Nav extends HTMLElement {
+  constructor() {
+    super();
+    this.navLinks = [
+      {
+        name: 'Home',
+        url: '/'
+      }, {
+        name: 'Events',
+        url: '/events'
+      }, {
+        name: 'Media',
+        url: '/media'
+      },
+      {
+        name: 'Merchendise',
+        url: '/merchendise'
+      },
+      {
+        name: 'The Crew',
+        url: '/the_crew'
+      }
+    ];
+  }
+
+  connectedCallback() {
+    const { navLinks } = this;
+
+    this.innerHTML = `
+      <nav class="flex justify-center p-4">
+        ${
```

Comment:
What are your thoughts on using `<ul>` for a navigation?  I always wondered if that was more or less semantic than a list of `<a>` tags.  I've always used `<ul>` but I'm not sure if conventions have evolved since then (a long time ago when I first started web dev :D )